ui/src/app/components/object-detail/object-detail.component.scss

```diff
@@ -156,7 +156,7 @@
       background-color: inherit;
       border: none;
       .edit-icon {
-        color: black;
+        color: #3367d6;
```

Comment:
Pencil icon should be blue in the top where we set spanner instance also

Reply:
Updated